resources/s3-objects.go

```diff
@@ -102,7 +102,8 @@ func (e *S3Object) Properties() types.Properties {
 		Set("Bucket", e.bucket).
 		Set("Key", e.key).
 		Set("VersionID", e.versionID).
-		Set("IsLatest", e.latest)
+		Set("IsLatest", e.latest).
+		Set("CreationDate", e.CreationDate())
```

Comment:
I'm fairly new to Go so was not sure if it should be `e.CreationDate()` or `e.CreationDate`. I would be grateful of a poke in the right direction 😄

Reply:
According to the [docs](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/s3-example-basic-bucket-operations.html#:~:text=creation%20date%20of%20each%20bucket), it would appear that I should have gone with `e.CreationDate` - will update.